src/__fixtures__/real-world/vendor-extensions.json

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
Could you make this example a bit smaller? Could be just a few lines of schema to showcase extensions.

Reply:
I have changed it and split it out of `real-world` its own section